ui/job-view/pushes/JobButton.jsx

```diff
@@ -5,17 +5,17 @@ import { faStar as faStarRegular } from '@fortawesome/free-regular-svg-icons';
 import { faStar as faStarSolid } from '@fortawesome/free-solid-svg-icons';
 
 import { getBtnClass, findJobInstance } from '../../helpers/job';
-import { getSelectedJobId, getUrlParam } from '../../helpers/location';
+import { getUrlParam } from '../../helpers/location';
 
 export default class JobButtonComponent extends React.Component {
   constructor(props) {
     super(props);
 
     const { job } = this.props;
-    const urlSelectedJob = getUrlParam('selectedJob');
+    const urlSelectedTaskRun = getUrlParam('selectedTaskRun');
```

Comment:
So this will replace support for the `selectedJob` param entirely? I'm wondering if it's worth keeping both, since one refers to a pk in our database and `selectedTaskRun` is different. Two ways to search for the same data?

Reply:
My inclination here is to allow it to work, but still switch them to selectedTask.  Otherwise, it will require a lot of funky logic internally to check which way they did it initially and maintain that throughout the code.

If you don't have a strong desire for me to maintain between the two, I'd like to do it this way.  But, as always, I'm open to ideas.  :)

Reply:
I think this is fine, was just wondering about the uses cases for being able to query by a jobId. :) Either way, not a blocker.